src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -428,7 +428,7 @@ private static void BindProperty(PropertyInfo property, object instance, IConfig
             }
             else
             {
-                if (isParentCollection && bindingPoint.Value is null)
+                if (isParentCollection && bindingPoint.Value is null && string.IsNullOrEmpty(configValue))
```

Comment:
Can we double check that an empty config value here will mean the same thing as null?  I didn't see any other place where we treated an empty string as an unspecified value.

Reply:
Chatted with Tarek.  Empty string is specifically the case we want to let through to create an instance.